generatedocs/data/actions/publishbookmark/examples.md

```diff
@@ -0,0 +1,27 @@
+### Example
+
+Publish bookmark previously created in the script with an id `bookmark1`.
```

Comment:
Maybe change to "Publish the bookmark with `id` "bookmark1" that was created earlier on in the script."?